lib/xboxrt/libc_extensions/direct.c

```diff
@@ -0,0 +1,108 @@
+// SPDX-License-Identifier: MIT
+
+// SPDX-FileCopyrightText: 2023 ExoSkye
+
+// Part of Microsoft CRT
+
+#include <errno.h>
+#include <string.h>
+#include <stdbool.h>
+#include <windows.h>
+
+// Made referencing https://www.digitalmars.com/rtl/direct.html, retrieved 2023-06-10, copyrighted 1999-2018 by Digital Mars
+
+#define MAXPATH 80
+#define MAXDRIVE 3
+
+/*
+ * These silently succeed, since they wouldn't have any effect on the program at all
+ */
+
+int _chdir(char* path) {
+    return 0;
+}
+
+int _chdrive(int drive) {
+    return 0;
+}
+
+/*
+ * The Xbox has no concept of current working directory, so these can't work
```

Comment:
Pretty sure it does?

If not, we could still implement it. After all, the libc is implemented in nxdk.

So while this wouldn't work for kernel functions (although it might, because they must also do relative or absolute access at some point - both of which could be remapped), it would still work for the libc or our winapi.

I'm fine with stubbing these for now, but I think there's room for improvement.

Reply:
I'm pretty sure it doesn't, based on what RadWolfie said and also that I can't find any kind of reference to it in the nxdk source tree currently

But it would be good to implement it eventually

Reply:
I have a feeling that implementing relative path support (if there really is none) should be done in a separate PR, so I may just leave these stubbed for now